mivs/configspec.ini

```diff
@@ -0,0 +1,67 @@
+
```

Comment:
If anyone is curious, Uber uses the ``ConfigObj`` library for configuration: https://configobj.readthedocs.org/

It's really great and definitely my recommendation for anytime you need to work with config files in Python.